packages/frontend/src/components/DataViz/config/PieChartConfiguration.tsx

```diff
@@ -18,7 +18,7 @@ export const PieChartConfiguration = ({
     const dispatch = useVizDispatch();
 
     const groupField = useVizSelector(
-        (state) => state.pieChartConfig.config?.fieldConfig?.x.reference,
+        (state) => state.pieChartConfig.config?.fieldConfig?.x?.reference,
```

Comment:
For the separate PR: add the `clearable` and `value={groupField?.reference ?? null}`